Documentation/project-docs/dogfooding.md

```diff
@@ -13,59 +13,16 @@ sure to consult this document often.
 1. Install CLI 2.0.0-alpha SDK
     - [Win 64-bit Latest](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/dotnet-dev-win-x64.latest.exe)
     - [macOS 64-bit Latest](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/dotnet-dev-osx-x64.latest.pkg)
-    - [Others](https://github.com/dotnet/cli/blob/master/README.md)
+    - [Others](https://github.com/dotnet/cli/blob/master/README.md#installers-and-binaries)
 
 ## Setup the project
 
 1. Create a new project
-    - Creat a new folder for your app
+    - Create a new folder for your app
+    - Ensure you have a "2.0.0-*" CLI installed on your path. You can check with `dotnet --info`.
     - Create project file by running `dotnet new`
 
-2. Add the CoreFX MyGet feed to your NuGet configuration.
-    - You can do this globally but we recommend not doing this as this might
-      affect other projects on your machine and you probably don't want that.
-    - Instead, add a `nuget.config` that is local to your project. You can
-      just put it next to the `.csproj` file.
-      See the [NuGet docs](https://docs.nuget.org/ndocs/consume-packages/configuring-nuget-behavior)
-      for details.
-
-    ```xml
-    <configuration>
-      <packageSources>
-        <add key="CoreFX" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
The NuGet.config file is simple, there's not need to change its contents:

https://github.com/dotnet/cli/tree/master/src/dotnet/commands/dotnet-new/CSharp_Console

It's the .csproj that is a little harder.  We add a template parameter that gets replaced during `dotnet new`:

https://github.com/dotnet/cli/blob/master/src/dotnet/commands/dotnet-new/Program.cs#L69-L77